manylinux/before_all.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 set -e
 
-yum-config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/cuda-rhel7.repo
+yum config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel8/x86_64/cuda-rhel8.repo
```

Comment:
Did the syntax change in rhel8?
* I was confused when looking up https://man7.org/linux/man-pages/man1/yum-config-manager.1.html (and seeing it was still `yum-config-manager`)
* Ended up doing a `docker run redhat/ubi8` to check the syntax and it is indeed `yum config-manager ...`

Reply:
It seemed to have changed - before I made this change I got a command not found error with manylinux_2_28.